homeassistant/components/advantage_air/binary_sensor.py

```diff
@@ -44,7 +48,7 @@ class AdvantageAirZoneFilter(AdvantageAirEntity, BinarySensorEntity):
     _attr_device_class = BinarySensorDeviceClass.PROBLEM
     _attr_entity_category = EntityCategory.DIAGNOSTIC
 
-    def __init__(self, instance, ac_key):
+    def __init__(self, instance: advantage_air, ac_key: int) -> None:
```

Comment:
ac_key is a string
https://github.com/home-assistant/core/blob/dev/tests/components/advantage_air/fixtures/getSystemData.json#L3